### PR TITLE
Update cattrs to 22.2.0

### PR DESCRIPTION
**Pull request by daniel-wer**
### Description:
- Update cattrs to 22.2.0 to enable and fix `Literal` structuring support in voxelytics. Also needed to make cattrs work for newer python versions, see [changelog](https://cattrs.readthedocs.io/en/stable/history.html).

### Todos:
Make sure to delete unnecessary points or to check all before merging:
 - [x] Updated Changelog

### CODE REVIEW

**Comment by daniel-wer** [comment]
@jstriebel Or would it be better to still allow the older versions? :thinking:

File: webknossos/pyproject.toml [hunk 1]
```diff
@@ -33,7 +33,7 @@ python = ">=3.7.1,<3.10"
 aiohttp = "^3.8.1"
 attrs = "^21.1.0"
 boltons = "~21.0.0"
-cattrs = "22.2.0"
+cattrs = ">=1.7.1,<=22.2.0"
```
Comment (jstriebel) [inline webknossos/pyproject.toml:36]: I think specifying the lower end is enough:
```suggestion
cattrs = ">=1.7.1"
```

Comment (daniel-wer) [reply webknossos/pyproject.toml:36]: Wouldn't that be susceptible to untested new versions and potential breaking changes, though?

**Review by jstriebel — APPROVED** [review]
:shipit:

**Pull request merged**